spec/models/article_search_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Article do
   describe ".text_search" do
-    before { 100.times { create :article } }
+    before { 10.times { create :article } }
```

Comment:
The goal with 100 articles in the search spec is to test with a sizeable set to try to isolate common issues that seem to arise with search. I understand it slows down the build but I'm not really worried about that at this point. Surfacing search issues is more important to me than speeding up the tests by a few seconds since they're already pretty fast. :-)